opencog/guile/SchemeSmobTV.cc

```diff
@@ -460,5 +460,32 @@ SCM SchemeSmob::ss_tv_get_value (SCM s)
 	return SCM_EOL;
 }
 
+/**
+ * Return the truth value mean
+ */
+SCM SchemeSmob::ss_tv_get_mean(SCM s)
+{
+	TruthValue *tv = verify_tv(s, "cog-tv-mean");
+	return scm_from_double(tv->getMean());
+}
```

Comment:
Well, the scheme code should be fixed up to do "the right thing" even for ITV. No one has used the ITV code before, so it wasn't supported.

The scheme wrappers are used by the language-learning code to store mutual information and counts in TV values, and I fear that TV merge strategies will screw these up.  You can't just merge MI unless you know what you're doing and why you're doing it. This is why I take a very dim view of doing the TV merge in C++ code -- it is almost guaranteed to do the wrong thing at the wrong time for the wrong reason.  This needs to be fixed.